testkit/src/main/scala/org/bitcoins/testkit/node/NodeUnitTest.scala

```diff
@@ -184,23 +205,26 @@ object NodeUnitTest extends P2PLogger {
 
   }
 
-  def destroySpvNode(spvNode: SpvNode)(
+  def destroyNode(node: Node)(
       implicit config: BitcoinSAppConfig,
       ec: ExecutionContext): Future[Unit] = {
-    val stopF = spvNode.stop()
-    stopF.flatMap(_ => ChainUnitTest.destroyHeaderTable())
+    node
+      .stop()
+      .flatMap(_ => ChainUnitTest.destroyHeaderTable())
```

Comment:
`ChainUnitTest.destroyAllTables()` rather than destroying each of them individually.